src/show/pages/private/CreateProject/Subviews/DeadlineStep/index.js

```diff
@@ -17,10 +17,13 @@ const DeadlineStep = ({ nextStep, previousStep, project, setProject }) => {
             <ReactDatePicker
               required
               className='flex justify-center text-center w-full '
-              selected={project?.deadline}
-              onChange={deadline =>
-                setProject({ ...project, deadline: deadline })
+              selected={
+                project.deadline ? new Date(project.deadline) : new Date()
               }
+              onChange={deadline => {
+                setProject({ ...project, deadline: deadline });
+                console.log(deadline);
```

Comment:
Remove console log and make it a one liner again.

Reply:
--------